apps/site/src/app/(home)/sections/Sponsors/Sponsors.tsx

```diff
@@ -3,6 +3,7 @@ import { getSponsors } from "./getSponsors";
 import styles from "./Sponsors.module.scss";
 import { client } from "@/lib/sanity/client";
 import imageUrlBuilder from "@sanity/image-url";
+import { Container } from "react-bootstrap";
```

Comment:
Chore: I forget if Next.js handles this well, but to be safe, we'll want to import just the individual component, i.e. `import Container from "react-bootstrap/Container"`
> Import individual components rather than the entire library. Doing so pulls in only the specific components that you use ([React Bootstrap, Importing Components](https://react-bootstrap.github.io/docs/getting-started/introduction#importing-components))